tests/avro/mock_registry.py

```diff
@@ -174,11 +176,18 @@ def __init__(self, port):
         self.server = None
         self.port = port
         self.daemon = True
+        self.started = Event()
 
     def run(self):
         self.server = MockServer(('127.0.0.1', self.port), ReqHandler)
+        self.started.set()
         self.server.serve_forever()
 
+    def start(self):
+        """ Start, and wait for server to be fully started, before returning. """
```

Comment:
nit: Per [PEP-257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), docstrings don't need space before/after the sentence.